test/e2e/data/kubetest/conformance-fast.yaml

```diff
@@ -1,8 +1,9 @@
 ginkgo.focus: \[Conformance\]
 ginkgo.skip: .*\[Serial\].*
 disable-log-dump: true
-ginkgo.progress: true
+ginkgo.show-node-evens: true
```

Comment:
nit: bumped into this before and here accidentally, spelling is incorrect for the new parameter, should be `--show-node-events`, see the deprecation warning below for ref: 
```
 Test Suite Failed
You're using deprecated Ginkgo functionality:
=============================================
  --progress is deprecated .  The functionality provided by --progress was confusing and is no longer needed.  Use --show-node-events instead to see node entry and exit events included in the timeline of failed and verbose specs.  Or you can run with -vv to always see all node events.  Lastly, --poll-progress-after and the PollProgressAfter decorator now provide a better mechanism for debugging specs that tend to get stuck. 
```

Reply:
Interesting copy&paste fail. Even in the PR description I've added the wrong parameter 😅 🤷🏻‍♂️

Reply:
Also forgot to change `-progress` inside our `Makefile` -- initially searched for `--progress` 🤦🏻‍♂️